BUILD.md

```diff
@@ -10,10 +10,10 @@
 
 Eclair supports deterministic builds for the eclair-core submodule, this is the 'core' of the eclair application
 and its artifact can be deterministically built achieving byte-to-byte equality for each build. This process helps
-to ensure that the artifact you're building is exactly the same as the one published in the release, please note
-that the deterministic build is limited to eclair-core and is also OS dependent, only unix based builds might be
-published in the release page. To check if your artifact is correct please check out the release page for the latest 
-published release.
+to ensure the artifact you're building is exactly the same as the one published in the release, please note
+that the deterministic build is limited to eclair-core and is OS dependent, to build the artifact deterministically
+you must use Ubuntu 18.10, JDK11.0.4 and maven 3.6.0. To check if your artifact is correct please check out the release 
```

Comment:
These versions will change over time (maven 3.6.0 files are not available anymore for example, and JDK11.0.6 has been released a few weeks ago). I propose instead: 
`...byte-to-byte equality for each build. To build the exact same artifacts that we release, you must use the build environment (OS, JDK, maven...) that we specify in our release notes.`